cronet/src/test/java/io/grpc/cronet/CronetClientStreamTest.java

```diff
@@ -207,26 +202,22 @@ public void write() {
   }
 
   private static List<Map.Entry<String, String>> responseHeader(String status) {
-    Map<String, String> headers = new HashMap<String, String>();
+    Map<String, String> headers = new HashMap<>();
     headers.put(":status", status);
     headers.put("content-type", "application/grpc");
     headers.put("test-key", "test-value");
-    List<Map.Entry<String, String>> headerList = new ArrayList<Map.Entry<String, String>>(3);
-    for (Map.Entry<String, String> entry : headers.entrySet()) {
-      headerList.add(entry);
-    }
+    List<Map.Entry<String, String>> headerList = new ArrayList<>(3);
```

Comment:
You could go even further, right?
```java
return new ArrayList<>(headers.entrySet());
```

Reply:
Yup, Updated
This are results of just depending on IDE suggestions :p